### DISCUSSION TITLE
Utility Rate  API

### DISCUSSION THREAD

**Discussion by supremepromise** [community/community#65419] [Programming Help]
What interfaces are available to view the electricity tariff rate plans of electric companies across the United States (free or charged)?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @supremepromise ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
